lib/background/actions/calendar_action_handler.dart

```diff
@@ -59,20 +60,23 @@ class CalendarActionHandler implements ActionHandler {
 
   @override
   Future<TimelineActionResponse> _handleRemoveEventAction(
-      TimelinePin pin,) async {
+    TimelinePin pin,
+  ) async {
     Future.microtask(() async {
       await _dao.setSyncAction(pin.itemId, NextSyncAction.Delete);
       await _watchSyncer.syncPinDatabaseWithWatch();
     });
 
     return TimelineActionResponse(true, attributes: [
-      TimelineAttribute.subtitle("Removed"),
+      TimelineAttribute.subtitle(tr.timelineAttribute.subtitle.removed),
```

Comment:
what does `tr` stand for?

Reply:
Translation, I guess? I took it from easy_localization package, they are using the same name. Should I rename it?